src/services/shopify-client.doubles.ts

```diff
@@ -0,0 +1,56 @@
+import {
+  ShopifyClient,
+  SubscriberListState,
+  ConnectedShop,
+  ShopifyErrorResult,
+} from './shopify-client';
+import { timeout } from '../utils';
+import { Result } from '../doppler-types';
+
+const oneShop = [
+  {
+    shopName: 'myshop.com',
+    synchronization_date: new Date('2017-12-17'),
+    list: {
+      name: 'MyList',
+      id: 1251,
+      amountSubscribers: 2,
+      state: SubscriberListState.ready,
+    },
+  },
+];
+
+/*const twoShops = [
+  {
+    shopName: 'myshop.com',
+    synchronization_date: new Date('2017-12-17'),
+    list: {
+      name: 'MyList',
+      id: 1251,
+      amountSubscribers: 2,
+      state: SubscriberListState.ready,
+    },
+  },
+  {
+    shopName: 'myshop2.com',
+    synchronization_date: new Date('2017-12-17'),
+    list: {
+      name: 'MyList',
+      id: 1251,
+      amountSubscribers: 2,
+      state: SubscriberListState.ready,
+    },
+  },
+];*/
+
+export class HardcodedShopifyClient implements ShopifyClient {
+  public async getShopifyData(): Promise<Result<ConnectedShop[], ShopifyErrorResult>> {
+    console.log('getShopifyData');
+    await timeout(1500);
+    //return {success: false, message: 'Some random error'};
```

Comment:
need all the comments? what do you think @andresmoschini

Reply:
I've added this again, at @andresmoschini request. For quick testing purposes.

Reply:
These comments are really useful for testing weird scenarios in the local environment.

Reply:
thanks for answer!